### PR TITLE
Update PixelShuffler.py

### PR DESCRIPTION
**Pull request by L4Z3RC47**
Keras 2.2.2 moves normalize_data_format to keras.backend from conv_utils

### CODE REVIEW

**Comment by torzdf** [comment]
requirements.txt also will need updating, and testing for Keras version bump everywhere it is used will need to be performed.
